frontend/src/components/shared/AssessmentTreeNavigation.tsx

```
@@ -632,16 +638,90 @@ const AssessmentTreeNavigation: React.FC<AssessmentTreeNavigationProps> = ({
+                                                <SidebarMenuSubButton
+                                                  onClick={() => {
+                                                    if (premiumStatus) {
+                                                      router.push(`/assess/${projectId}/crc?category=${encodeURIComponent(cat)}`);
+                                                    } else {
+                                                      openSubscriptionModal("Unlock Premium to Access Compliance Readiness Controls", "Upgrade to premium to unlock this feature and many more advanced capabilities.");
+                                                    }
+                                                  }}
+                                                  className="h-8 px-2 group/cat"
+                                                >
+                                                   <div
+                                                     role="button"
```

Biome (2.4.7)

[error] The elements with this role can be changed to the following elements:
<button>

(lint/a11y/useSemanticElements)